src/cli.ts

```diff
@@ -143,8 +143,11 @@ const run = defineCommand({
     },
     ...operationArgs,
   },
-  run: async ({ args }) => {
-    await runScript(args.name, args);
+  run: async ({ args, rawArgs }) => {
+    await runScript(args.name, {
+      ...args,
+      args: rawArgs.slice(1),
```

Comment:
We probably have to only pass args after `--` otherwise ones like `--cwd` will be passthrough